pkg/sql/logictest/testdata/logic_test/crdb_internal_tenant

```diff
@@ -10,3 +10,505 @@ SELECT node_id, name FROM crdb_internal.leases ORDER BY name
+----
+
+query error pq: foo
+SELECT crdb_internal.force_error('', 'foo')
+
+query error pgcode FOOYAA pq: foo
+SELECT crdb_internal.force_error('FOOYAA', 'foo')
+
+query I
+select crdb_internal.force_retry(interval '0s')
+----
+0
+
+query error pq: crdb_internal.set_vmodule\(\): syntax error: expect comma-separated list of filename=N
+select crdb_internal.set_vmodule('not anything reasonable')
+
+query I
+select crdb_internal.set_vmodule('doesntexist=2,butitsok=4')
+----
+0
+
+query I
+select crdb_internal.set_vmodule('')
+----
+0
+
+query T
+select regexp_replace(crdb_internal.node_executable_version()::string, '(-\d+)?$', '');
+----
+20.2
+
+query ITTT colnames
+select node_id, component, field, regexp_replace(regexp_replace(value, '^\d+$', '<port>'), e':\\d+', ':<port>') as value from crdb_internal.node_runtime_info
+----
+node_id  component  field   value
+0        DB         URL     postgresql://root@127.0.0.1:<port>?sslcert=test_certs%2Fclient.root.crt&sslkey=test_certs%2Fclient.root.key&sslmode=verify-full&sslrootcert=test_certs%2Fca.crt
+0        DB         Scheme  postgresql
+0        DB         User    root
+0        DB         Host    127.0.0.1
+0        DB         Port    <port>
+0        DB         URI     /?sslcert=test_certs%2Fclient.root.crt&sslkey=test_certs%2Fclient.root.key&sslmode=verify-full&sslrootcert=test_certs%2Fca.crt
+0        UI         URL     https://127.0.0.1:<port>
+0        UI         Scheme  https
+0        UI         User    ·
+0        UI         Host    127.0.0.1
+0        UI         Port    <port>
+0        UI         URI     /
+
+statement error unsupported in multi-tenancy mode
+SELECT node_id, network, regexp_replace(address, '\d+$', '<port>') as address, attrs, locality, regexp_replace(server_version, '^\d+\.\d+(-\d+)?$', '<server_version>') as server_version FROM crdb_internal.gossip_nodes WHERE node_id = 1
+
+statement error unsupported in multi-tenancy mode
+SELECT node_id, epoch, regexp_replace(expiration, '^\d+\.\d+,\d+$', '<timestamp>') as expiration, draining, decommissioning, membership FROM crdb_internal.gossip_liveness WHERE node_id = 1
+
+statement error unsupported in multi-tenancy mode
+SELECT node_id, network, regexp_replace(address, '\d+$', '<port>') as address, attrs, locality, regexp_replace(server_version, '^\d+\.\d+(-\d+)?$', '<server_version>') as server_version, regexp_replace(go_version, '^go.+$', '<go_version>') as go_version
+FROM crdb_internal.kv_node_status WHERE node_id = 1
+
+statement error unsupported in multi-tenancy mode
+SELECT node_id, store_id, attrs, used
+FROM crdb_internal.kv_store_status WHERE node_id = 1
+
+statement ok
+CREATE TABLE foo (a INT PRIMARY KEY, INDEX idx(a)); INSERT INTO foo VALUES(1)
+
+statement error unsupported in multi-tenancy mode
+ALTER TABLE foo SPLIT AT VALUES(2)
+
+# Cluster ID is unset for tenants.
+query B
+select crdb_internal.cluster_id() != '00000000-0000-0000-0000-000000000000' FROM foo
+----
+false
```

Comment:
@tbg is this expected?

Reply:
Yes, it is. The ClusterID is primarily a KV concept. Tenants could learn it only from Gossip, but they don't since that has not been necessary.